targets/cpp-ue4/Plugins/PlayFab/Source/PlayFab/Public/Core/PlayFabCore.h

```diff
@@ -1,13 +0,0 @@
-#pragma once
```

Comment:
Wait... really? This isn't needed at all?

Reply:
As far as I can tell, the file is empty and isn't included anywhere. The plugin compiles fine without it.